tests/e2e/csi_snapshot_basic.go

```diff
@@ -4145,7 +4175,7 @@ var _ = ginkgo.Describe("[block-vanilla-snapshot] Volume Snapshot Basic Test", f
 			restoredpvclaims = append(restoredpvclaims, pvclaim2)
 
 			persistentvolume2, err := fpv.WaitForPVClaimBoundPhase(client, []*v1.PersistentVolumeClaim{pvclaim2},
-				framework.ClaimProvisionTimeout)
+				totalResizeWaitPeriod)
```

Comment:
can you add comments why we are waiting longer
also use `framework.ClaimProvisionTimeout * 2` instead of `totalResizeWaitPeriod` which may get changed based on requirements of resize related tests